src/pybind/mgr/volumes/module.py

```diff
@@ -212,7 +215,31 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'desc': "Cancel an pending or ongoing clone operation.",
             'perm': 'r'
         },
-
+        {
+            'cmd': 'nfs export create '
+            'name=type,type=CephString '
+            'name=fsname,type=CephString '
+            'name=binding,type=CephString '
+            'name=attach,type=CephString '
+            'name=readonly,type=CephBool,req=false '
+            'name=path,type=CephString,req=false ',
+            'desc': "Create a cephfs export",
+            'perm': 'rw'
+        },
+        {
+            'cmd': 'fs nfs export delete '
+                   'name=export_id,type=CephInt,req=true ',
+            'desc': "Delete a cephfs export",
+            'perm': 'rw'
+        },
+        {
+            'cmd': 'nfs cluster create '
+                   'name=type,type=CephString '
+                   'name=clusterid,type=CephString '
+                   'name=size,type=CephInt,req=false ',
```

Comment:
:+1: 

later, for adding a cephadm integration, you'll need a `'name=placement,type=CephString,req=false '` parameter